scripts/dbMaintenance/removeOrphanFiles.py

```diff
@@ -58,34 +62,42 @@
                     print("\t\t--stash: " + colName)
                 for refEntry in db[colName].find({"type": "fs"}):
                     filePath = os.path.normpath(os.path.join(localFolder, refEntry['link']))
-                    fileStatus = fileList.get(filePath)
-                    if fileStatus == None:
-                        refInfo = database + "." + modelId + "." + colName + ": " + refEntry["_id"]
-                        if dryRun:
-                            missing.append(refInfo);
-                        else:
-##### Upload missing files to FS and insert BSON #####
-                            fs = gridfs.GridFS(db, modelId + colPrefix)
-                            if colPrefix == ".history":
-                                toRepair = refEntry["_id"]
+                    if not [ x for x in ignoreDirs if filePath.find(x) + 1 ]:
```

Comment:
This logic is a bit complicated in a single line... took me a while to understand exactly what's going on even though I've guessed it from the context of this PR.

Would be good if we either "autocomment" it by putting it in a variable, something like this:
```suggestion
                    inIgnoreDir= bool([ x for x in ignoreDirs if filePath.find(x) + 1 ])
                    if not inIgnoreDir:
```
Or just write a comment on top of it